test/test_torch.py

```diff
@@ -11169,80 +11169,97 @@ def test_signbit_non_boolean_output(self, device, dtype):
             torch.signbit(t, out=out)
 
     def test_logical_any(self, device):
-        x = torch.zeros([2, 3, 400], dtype=torch.uint8, device=device)
+        for dtype in torch.testing.get_all_dtypes(include_half=True, include_bfloat16=False,
```

Comment:
Instead of enumerating dtypes here use the @dtypes decorator on the test

Reply:
Done.